packages/theme-chalk/src/input.scss

```
@@ -54,6 +54,8 @@
     position: relative;
     display: block;
     resize: vertical;
+    // #24721 Keep Firefox's caret in view after autosize temporarily hides overflow.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)